src/frontend/package.json

```diff
@@ -3,8 +3,9 @@
   "private": true,
   "dependencies": {
     "@apollo/react-hooks": "3.1.3",
-    "@material-ui/core": "4.9.8",
+    "@material-ui/core": "4.9.9",
     "@material-ui/icons": "4.9.1",
+    "@material-ui/lab": "4.0.0-alpha.48",
```

Comment:
This makes me nervous.  Can we not stick with stable components?

Reply:
> I also updated the package @material-ui/core so I can add+use the experimental package @material-ui/lab for Alerts

Reply:
What I'm saying is, I don't think we should be using this:

> This package hosts the incubator components that are not yet ready to move to core

If it's not ready for core, why are we taking it in our tree?  Anything that isn't stable is necessarily unstable, and changing, which means APIs will break.

Why can't we use `import Snackbar from '@material-ui/core/Snackbar';`?